website/docs/r/azure_event_hub_log_source.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_azure_event_hub_log_source"
+description: |-
+  Provides a Sumologic Azure Event Hub Log Source.
+---
+
+# sumologic_azure_event_hub_log_source
+Provides a [Sumologic Azure Event Hub Log Source][2].
+
+__IMPORTANT:__ The Azure Event Hub credentials are stored in plain-text in the state. This is a potential security issue.
+
+## Example Usage
+```hcl
+resource "sumologic_azure_event_hub_log_source" "terraform_azure_event_hub_log_source" {
+  name          = "Azure Event Hub Log Source"
+  description   = "My description"
+  category      = "azure/eventhub"
+  content_type  = "AzureEventHubLog"
+  collector_id  = "${sumologic_collector.collector.id}"
+  authentication {
+    type = "AzureEventHubAuthentication"
+	shared_access_policy_name = "%s"
+	shared_access_policy_key = "%s"
+  }
+  path {
+    type = "AzureEventHubPath"
+	namespace     = "%s"
+	event_hub_name = "%s"
+	consumer_group = "%s"
+    region = "%s"
+  }
+}
+resource "sumologic_collector" "collector" {
+  name        = "my-collector"
+  description = "Just testing this"
+}
+```
+
+## Argument reference
+
+In addition to the [Common Source Properties](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs#common-source-properties), the following arguments are supported:
+
+ - `content_type` - (Required) The content-type of the collected data. Details can be found in the [Sumologic documentation for hosted sources][1].
```

Comment:
We should be more prescriptive, `content_type` needs to be `AzureEventHubLog` for this source type